Jint/Native/Json/JsonInstance.cs

```diff
@@ -36,10 +38,93 @@ public void Configure()
             FastAddProperty("stringify", new ClrFunctionInstance(Engine, Stringify, 3), true, false, true);
         }
 
+        private JsValue AbstractWalkOperation(ObjectInstance thisObject, string prop)
+        {
+            JsValue value = thisObject.Get(prop);
+            if (value.IsObject())
+            {
+                var valueAsObject = value.AsObject();
+                if (valueAsObject.Class == "Array")
+                {
+                    var valAsArray = value.AsArray();
+                    var i = 0;
+                    var arrLen = valAsArray.GetLength();
+                    while (i < arrLen)
+                    {
+                        var newValue = AbstractWalkOperation(valAsArray, i.ToString());
```

Comment:
ideally always call TypeConverter.ToString(i), it has cached lookups for strings, this i.ToString() has been repeated three times here